code/game/objects/items/pinpointer.dm

```diff
@@ -158,6 +163,27 @@
 	if(!target) //target can be set to null from above code, or elsewhere
 		active = FALSE
 
+/obj/item/pinpointer/crew/prox //Weaker version of crew monitor primarily for EMT
+	name = "proximity crew pinpointer"
+	desc = "A handheld tracking device that displays its proximity to crew suit sensors."
+	icon_state = "pinpointer_crewprox"
+	custom_price = 300
+
+/obj/item/pinpointer/crew/prox/update_direction_icon(here, there)
+	var/size = ""
+	if(here == there)
+		size = "small"
+	else
+		switch(get_dist(here, there))
+			if(1 to 4)
+				size = "xtrlarge"
+			if(5 to 16)
+				size = "large"
+//			if(17 to 28)
+//				use normal pinion
```

Comment:
As noted in https://github.com/tgstation/tgstation/pull/48418#issuecomment-569315970, I put this here to help understand what happens rather than inferring that the normal one is `pinondirect` without a suffix.

Reply:
```DM
//17 to 28 uses the normal pinion hence why they're omitted
```

looks like commented code

Reply:
good point